spec/appliance_console/cli_spec.rb

```diff
@@ -227,6 +227,48 @@
     end
   end
 
+  context "#set_server_state" do
+    let(:evm_service) { double("evmserved") }
+    before do
+      allow(LinuxAdmin::Service).to receive(:new).with("evmserverd").and_return(evm_service)
+    end
+    it "state is status" do
+      expect(subject).to receive(:say)
+      expect(evm_service).to receive(:running?).and_return(true)
+      subject.parse(%w(--server status)).run
+    end
+
+    it "state is start" do
+      expect(subject).to receive(:say)
+      expect(evm_service).to receive(:running?).and_return(true)
+      expect(evm_service).to receive(:start)
+      evm_service.start
```

Comment:
I can't test this without double with
``ruby
 context "#set_server_state" do
    before do
      allow(LinuxAdmin::Service).to receive(:new).with("evmserverd")
    end

    it "state is status" do
      expect(LinuxAdmin::Service).to receive(:new).with("evmserverd")
      expect_any_instance_of(LinuxAdmin::Service).to receive(:status).and_return(true)
      subject.parse(%w(--server status)).run
    end
```

throw the error
NoMethodError:
       undefined method `running?' for nil:NilClass

I don't know how test this kind of service without double